### PR TITLE
[Accuracy diff No.127] Fix accuracy diff for paddle.nn.functional.sigmoid_focal_loss API

### PR DESCRIPTION
**Pull request by NKNaN**
修改 config_analyzer 在 sigmoid_focal_loss 生成label时只生成 0 或 1 的值。并且由于paddle在计算时调用了 sigmoid_cross_entropy_with_logits，这部分反向计算不算 label_grad，所以 sigmoid_focal_loss 也不能检查 label_grad。

回测结果：
image

### CODE REVIEW

**Comment by Cutelemon6** [comment]
@cangtianhuang 加入到 grad_diff 的配置无需从原处删除？
